common/utils/src/main/resources/error/error-classes.json

```
@@ -2333,6 +2357,12 @@
     ],
     "sqlState" : "428C4"
   },
+  "UNRECOGNIZED_PARAMETER_NAME" : {
+    "message" : [
+      "Cannot invoke function <functionName> because the function call included a named argument reference for the argument named <argumentName>, but this function does not include any signature containing an argument with this name. Here are some potential candidates: \n<arguments>"
```

```suggestion
      "Cannot invoke function <functionName> because the function call included a named argument reference for the argument named <argumentName>, but this function does not include any signature containing an argument with this name. Did you mean one of the following? [<proposal>]."
```
Let's make it aligned with other `WITH_SUGGESTION` error classes.